### PR TITLE
Added a flag to specify OCI annotation namespace

### PR DESCRIPTION
**Pull request by peusebiu**
added --annotations-namespace arg

Signed-off-by: Petu Eusebiu <peusebiu@cisco.com>

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -1,6 +1,16 @@
 package stacker
 
+import "fmt"
+
 const (
-	GitVersionAnnotation      = "com.cisco.stacker.git_version"
-	StackerContentsAnnotation = "com.cisco.stacker.stacker_yaml"
+	GitVersionAnnotation      = "com.%s.stacker.git_version"
```
Comment (rchincha) [inline api.go:6]: remove the "com." as well

For example, this would "io.zotregistry.stacker.git_version"

Comment (rchincha) [reply api.go:6]: Thinking about this some more. Do we want to just default this to "io.project-stacker.stacker.git_version"

Comment (rchincha) [reply api.go:6]: "io.stackeroci.stacker.git_version"

Comment (peusebiu) [reply api.go:6]: ok, sure

File: squashfs/verity.go [hunk 2]
```diff
@@ -81,7 +81,7 @@ import (
 	"golang.org/x/sys/unix"
 )
 
-const VerityRootHashAnnotation = "com.cisco.stacker.squashfs_verity_root_hash"
+const VerityRootHashAnnotation = "io.stackeroci.stacker.squashfs_verity_root_hash"
```
[the remark this answers is not in the document]
Comment (andaaron) [reply squashfs/verity.go:84]: @hallyn what do you think?

[the remark this answers is not in the document]
Comment (hallyn) [reply squashfs/verity.go:84]: It probably shouldn't be configurable, as that will make images from different stacker builds non-interoperable, right?

[the remark this answers is not in the document]
Comment (andaaron) [reply squashfs/verity.go:84]: Ok, let's mark this as resolved.

**Pull request merged**